scripts/jenkins-test

```diff
@@ -26,16 +25,24 @@ export SPARK_DRIVER_MEMORY=8g
 
 pushd "$ADAM_TMP_DIR"
 
+
 if [[ $HADOOP_VERSION == "1.0.4" ]]; then
```

Comment:
Maybe `echo $HADOOP_VERSION | egrep '^1.0.'` would be more general, in case, we use e.g. `1.0.4` as the hadoop test version?